bc/users/urls.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import views as auth_views
+from django.urls import path
+
+from bc.core.utils.network import ratelimiter_unsafe_10_per_m
+
+urlpatterns = [
+    # Sign in page
+    path(
+        "sign-in/",
+        ratelimiter_unsafe_10_per_m(
+            auth_views.LoginView.as_view(
+                **{
+                    "template_name": "register/login.html",
+                    "extra_context": {"private": False},
```

Comment:
We're not using this yet in our base template, are we? If not, we should either remove it from here or add it to the base template. I think we can probably just remove it here and we won't have private pages in this project?

Reply:
No, We're not using the "private" attribute in the base.html. I'll remove it.